### PR TITLE
Add ENCRYPTION_KEYPAIRS to settings

### PR DESCRIPTION
**Pull request by gregorywong**
This PR introduces a new configuration variable `ENCRYPTION_KEYPAIRS` to the settings.
In addition to allowing the user to set it, this enhancement will automatically set it for them if both `KEY_FILE` and `CERT_FILE` are defined.

When handling **encrypted assertions**, [encryption_keypairs](https://pysaml2.readthedocs.io/en/latest/howto/config.html#encryption-keypairs) must be set in order for saml2 to properly decrypt them. Unfortunately, this is not fully documented, but it is a known issue noted by other users of pysaml2 (or libraries requiring it):
https://github.com/IdentityPython/pysaml2/issues/346
https://github.com/IdentityPython/djangosaml2/issues/22

This enhancement ensures that encrypted assertions can be handled successfully, while keeping the configuration settings DRY (i.e., no need to include `CERT_FILE` and `KEY_FILE` twice).

### CODE REVIEW

**Comment by gregorywong** [comment]
Tagging @mostafa for review. Thanks!

File: README.md [hunk 1]
```diff
@@ -111,6 +111,14 @@ python setup.py install
         'METADATA_LOCAL_FILE_PATH': '[The metadata configuration file path]',
         'KEY_FILE': '[The key file path]',
         'CERT_FILE': '[The certificate file path]',
+
```
Comment (mostafa) [inline README.md:114]: Please add these to the [module settings](https://github.com/grafana/django-saml2-auth?tab=readme-ov-file#module-settings) below as well.

**Review by mostafa — COMMENTED** [review]
@gregorywong 
Thanks for your contribution! LGTM, but please consider fixing the comment before I merge.

File: README.md (hunk 1 above)
Comment (gregorywong) [reply README.md:114]: @mostafa I've updated the module settings!

**Review by mostafa — APPROVED** [review]
:tada:

**Pull request merged**